### ISSUE TITLE
hook_views_handlers() removed. Add change record.

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#2819]
It looks like `hook_views_handlers()` was removed from backdrop, but we didn't add a change record.

At the very least we should state that this is not needed in Backdrop, and why.

**Comment by BWPanda** [comment]
> At the very least we should state that this is not needed in Backdrop, and why.

Yes, why is that? I'm trying to port a D7 module that implements this hook, and now I'm stuck not know what to do...

**Comment by indigoxela** [comment]
That hook was already deprecated in D7: https://www.drupal.org/project/views/issues/856186
